howdoi/howdoi.py

```diff
@@ -476,8 +513,24 @@ def get_parser():
                         action='store_true')
     parser.add_argument('-e', '--engine', help='change search engine for this query only (google, bing, duckduckgo)',
                         dest='search_engine', nargs="?", default='google')
+    parser.add_argument('--plugin', help='query a specific plugin in the plugins folder', type=str)
     return parser
 
+def parse_plugin_from_filepath(path):
+    try:
+        with open(path) as json_file:
+            plugin = {}
+            json_file_content = json.load(json_file)
+            plugin['url'] = json_file_content['url']
+            plugin['selector'] = json_file_content['selector']
+            return plugin
```

Comment:
Returning a dict is good. We can consider the dict to be the real "plugin", and the file to be an expression of it. Maybe other filetypes are possible.